### DISCUSSION TITLE
White List when below GP Threshold

### DISCUSSION THREAD

**Discussion by Castro3halo** [runelite/runelite#15527] [Ideas]
Being able to see bones etc. That are below the GP threshold without making them highlighted items at god wars, or blacklisting half the items in the game.

**Comment by mariosm1** [comment]
I'm not sure if this is what you meant, but there's a "Hide under value" option in the ground items plugin.
![Capture](https://user-images.githubusercontent.com/32443983/185723971-4be52e16-b382-4d4c-be39-4e1509e53850.PNG)

**Reply by Castro3halo** [reply]
No I mean a white list of items that will still show even when that is set at say 1k. 

E.g. Bones at godwars will still show up as a "normal" low value drop despite being under the 1k threshold if they are whitelisted.
